x-pack/plugins/fleet/public/components/agent_enrollment_flyout/confirm_agent_enrollment.tsx

```diff
@@ -134,17 +136,31 @@ export const ConfirmAgentEnrollment: React.FunctionComponent<Props> = ({
   if (showLoading && !agentCount) {
     return (
       <>
-        <EuiCallOut
-          size="m"
-          color="primary"
-          iconType={EuiLoadingSpinner}
-          title={
-            <FormattedMessage
-              id="xpack.fleet.agentEnrollment.loading.listening"
-              defaultMessage="Listening for agent..."
-            />
-          }
-        />
+        {isCsp ? (
+          <EuiCallOut
+            size="m"
+            color="primary"
+            iconType={EuiLoadingSpinner}
+            title={
+              <FormattedMessage
+                id="xpack.fleet.agentEnrollment.loading.listeningcsp"
+                defaultMessage="Listening for agent... this can take several minutes"
```

Comment:
Can we avoid the if/else logic completely? Doesn't `this can take several minutes` apply to any agent enrollment? If we believe this is a UX improvement, should we enable any integration benefit from it? I totally get that it's much easier to change smth that Cloud Security owns, but it leads to these UX differences which are hard to explain from the user's standpoint. Wdyt  @tinnytintin10, @animehart @juliaElastic ?

Reply:
Normally it shouldn't take that long to enroll an agent, but I see no harm in showing that for all cases.

Reply:
same here. maybe just wait for Tinsae's reply before we proceed

Reply:
@maxcold to Julie's point, normally agents don't take long to check-in as they are getting deployed on already running infrastructure. However, in our case, we're also provisioning the infrastructure (if a user uses the IaC option). It's with this in mind that I suggest this only gets applied in our case. Let's leave the decision to apply this more broadly with the fleet product team. 

cc @amitkanfer

Reply:
@animehart to not block the change, let's maybe move the if logic inside `title` and move forward. Another improvement might be to change the `isCsp` to smth like `isLongEnrollment` to make it more generic.